editor.planx.uk/src/ui/editor/ColorPicker.tsx

```diff
@@ -2,13 +2,16 @@ import Box, { BoxProps } from "@mui/material/Box";
 import ButtonBase, { ButtonBaseProps } from "@mui/material/ButtonBase";
 import { styled } from "@mui/material/styles";
 import Typography from "@mui/material/Typography";
+import { ErrorMessage } from "formik";
 import React, { useState } from "react";
 import { ChromePicker, ColorChangeHandler } from "react-color";
+import ErrorWrapper from "ui/shared/ErrorWrapper";
 
 export interface Props {
   label?: string;
   inline?: boolean;
   color?: string;
+  errorMessage?: string | undefined;
```

Comment:
```suggestion
  errorMessage?: string;
```

nit: This is the same thing but a bit more terse and matches our code style. The `?` following the property name indicates that this property can be `undefined`.